api/src/main/resources/liquibase.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet id="20231103-add-authentication-event-log" author="mseaton">
+        <preConditions onFail="MARK_RAN">
+            <not><tableExists tableName="authentication_event_log"/></not>
+        </preConditions>
+        <createTable tableName="authentication_event_log">
+            <column name="login_id" type="char(36)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="event_datetime" type="datetime">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ip_address" type="varchar(40)"/>
+            <column name="http_session_id" type="varchar(32)"/>
+            <column name="event_type" type="varchar(50)"/>
+            <column name="username" type="varchar(50)"/>
+            <column name="user_id" type="int"/>
+            <column name="scheme_id" type="varchar(50)"/>
+        </createTable>
+    </changeSet>
```

Comment:
This adds a table to the data model that we can use to track user authentication events (log ins, log outs, session expiration, whether 2FA was used to log in, etc)